src/components/Header/Logo.js

```diff
@@ -23,7 +24,7 @@ const Tagline = styled.p`
   margin: 0;
   padding: 0;
   color: ${colorBlack};
-  font-size: ${fontSmall};
+  font-size: 1.2rem;
```

Comment:
Why did you use straight values?

Reply:
It is more part of logo than regular page layout. So it has custom value to just look good next to logo.